From 034ac41561409ffc7836683331b6b94a641436e4 Mon Sep 17 00:00:00 2001
From: Tim Orling <tim.orling@konsulko.com>
Date: Mon, 28 Feb 2022 09:41:02 -0800
Subject: [PATCH] setup.py: switch from distutils to setuptools

In Python 3.10 'distutils' is deprecated with removal slated for Python
3.12. Switch from 'distutils.core' to 'setuptools'.

This also allows for a 'wheel' binary archive format to be built with
'setup.py bdist_wheel'.

Upstream-Status: Backport [cf9eafdd165347f86bb4a2534c41fd805db52322]

Signed-off-by: Tim Orling <tim.orling@konsulko.com>

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 9c12180..3ba5195 100644
--- a/setup.py
+++ b/setup.py
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
 
-from distutils.core import setup
+from setuptools import setup
 
 setup(name='python-meh', version='0.50.1',
       description='Python module for handling exceptions',
